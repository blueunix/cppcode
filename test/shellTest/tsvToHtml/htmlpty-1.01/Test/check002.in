<html>
<body>
<ul>
<li>the <a name=5
href="http://info.cern.ch/hypertext/WWW/Addressing/Addressing.html">URL 
(Uniform Resource Locator) specification</A>
</ul>
<a
		  name=5
		  name=5
		  name=5
		  name=5
		  name=5
		  name=5
		  name=5
		  name=5
		  name=5
		  name=5
		  name=5
		  name=5
		  name=5
href="http://info.cern.ch/hypertext/WWW/Addressing/VeryLongAddressing.html">
anchored
text
</a>

</body>
</html>
