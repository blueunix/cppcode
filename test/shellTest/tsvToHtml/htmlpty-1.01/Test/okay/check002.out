<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">

<HTML>
    <HEAD>
        <TITLE>
            <!-- Please supply a descriptive title here -->
        </TITLE>
        <!-- Please supply a correct e-mail address here -->
        <LINK REV="made" HREF="mailto:beebe@sunrise.math.utah.edu">
    </HEAD>
    <BODY>
        <UL>
            <LI>
                the
                <A NAME=5
                    HREF="http://info.cern.ch/hypertext/WWW/Addressing/Addressing.html">
                    URL (Uniform Resource Locator) specification
                </A>
            </LI>
        </UL>
        <A NAME=5 NAME=5 NAME=5 NAME=5 NAME=5 NAME=5 NAME=5 NAME=5
            NAME=5 NAME=5 NAME=5 NAME=5 NAME=5
            HREF="http://info.cern.ch/hypertext/WWW/Addressing/VeryLongAddressing.html">
            anchored text
        </A>
    </BODY>
</HTML>
